borg/archiver.py

```diff
@@ -1399,7 +1440,7 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('location', metavar='ARCHIVE',
                                type=location_validator(archive=True),
                                help='name of archive to create (must be also a valid directory name)')
-        subparser.add_argument('paths', metavar='PATH', nargs='+', type=str,
+        subparser.add_argument('paths', metavar='PATH', nargs='*', type=str,
```

Comment:
@enkore ok, that sounds good.

@leo-b i don't understand why your argparse hack works (arg is not being used in super call, so what does it do?). guess we rather don't want the other approach (monkeypatching it). Didn't yet look into argparse code.